test/data/test-app-regular/index.js

```diff
@@ -1,6 +1,7 @@
-var electron = require('electron');
-var path     = require('path');
-var dialog   = electron.dialog;
+var electron   = require('electron');
+var path       = require('path');
+var dialog     = electron.dialog;
+var remoteMain = require('@electron/remote/main');
```

Comment:
I suspect that this won't work with older Electron versions. Could you please test the changes with Electron 11 to see if we don't break things?